UI/obs-app.cpp

```diff
@@ -736,15 +736,21 @@ bool OBSApp::SetTheme(std::string name, std::string path)
 
 bool OBSApp::InitTheme()
 {
-	const char *themeName = config_get_string(globalConfig, "General",
+	const char *fallbackName = config_get_string(globalConfig, "General",
 			"Theme");
 
-	if (!themeName)
-		themeName = "Default";
+	if (!fallbackName)
+		fallbackName = "Default";
+
+	const char *themeName = config_get_string(globalConfig, "General",
+			"CustomTheme");
 
-	stringstream t;
-	t << themeName;
-	return SetTheme(t.str());
+	if (themeName && SetTheme(themeName)) {
+		if (SetTheme(themeName)) {
```

Comment:
Oh right. The cleaner code I got given didn't have the &&, but looks like I forgot to remove the inner if statement.